client/lib/happychat/connection.js

```diff
@@ -9,7 +9,7 @@ import { v4 as uuid } from 'uuid';
 /**
```

Comment:
Shouldn't this file also be moved to `client/extensions/happychat`?

Reply:
Indeed! Fixed in 148d8df